turbo.json

```diff
@@ -8,6 +8,7 @@
       "outputs": []
     },
     "dev": {
+      "dependsOn": ["@webstudio-is/sdk#build"],
```

Comment:
I think it shouldn't pre-build it in dev. Instead, it should work with the source code.